react/components/form/ImageUploader/ErrorAlert.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react'
+import { Alert } from 'vtex.styleguide'
+
+interface Props {
+  message: string
+}
+
+const ErrorAlert: React.SFC<Props> = ({ message }) => {
+  const [isVisible, setIsVisible] = useState<boolean>(true)
```

Comment:
I was about to comment "classy 🎩", but it's actually the opposite. Pun intended.

Reply:
hooky